### ISSUE TITLE
Unify QuantEcon.py and QuantEcon.jl APIs?

### ISSUE THREAD

**Issue by Smit-create** [QuantEcon/QuantEcon.py#663]
One of the APIs that differ in `julia` and `python` versions is: `tauchen`. 
Julia version is: [Doc1](http://quantecon.github.io/QuantEcon.jl/latest/api/QuantEcon.html#QuantEcon.tauchen-Union{Tuple{T3},%20Tuple{T2},%20Tuple{T1},%20Tuple{Integer,%20T1,%20T2},%20Tuple{Integer,%20T1,%20T2,%20Any},%20Tuple{Integer,%20T1,%20T2,%20Any,%20T3}}%20where%20{T1%3C:Real,%20T2%3C:Real,%20T3%3C:Real})
Python version is: [Doc2](https://github.com/QuantEcon/QuantEcon.py/blob/6ffbe570008979e22b3156dd77745990b53ac74d/quantecon/markov/approximation.py#L138)

We can try to unify such APIs that differ in `julia` and `python`(like name of function arguments, position of parameters, etc.) which will help the users in migrating from one language to the other easily.

**Comment by jstac** [comment]
Yes, that would be great @Smit-create .  The `tauchen` one has always bothered me.  This one might need unifying as well:

http://quantecon.github.io/QuantEcon.jl/latest/api/QuantEcon.html#QuantEcon.rouwenhorst

The only issue is that these are breaking changes, so we would need to coordinate changes on the lectures and any other downstream code (e.g., the DP textbook).

**Comment by oyamad** [comment]
See also #363.

**Comment by Smit-create** [comment]
One another difference:
Python: `markov_chain_object.P`
Julia: `markov_chain_object.p`

**Comment by marcdelabarrera** [comment]
Can you make the UserWarning to appear only once per session?
